build_tools/benchmarks/common/common_arguments.py

```diff
@@ -152,6 +152,11 @@ def __init__(self, *args, **kwargs):
         "for). In that case, no --benchmark_repetitions flag will be passed."
         " If not specified, a --benchmark_repetitions will be passed "
         "instead.")
+    self.add_argument(
+        "--disable_compatible_filter",
```

Comment:
Yeah I would flip this so the flag is `--filter_compatible` or `--compatible-only`. Then someone has to take explicit action to have the tool not run the benchmarks it just told it to run. It also has the advantage of using positive booleans, which are generally slightly easier to reason about (rather than thinking about `disable_compatible_filter` is `False` and dealing with the double negative)

Reply:
Done.